setup.py

```diff
@@ -21,7 +21,7 @@ def package_files(directory):
     description='A machine learning model to tag grants',
     packages=find_packages(),
     include_package_data=True,
-    version='2020.07.1',
+    version='2020.07.2',
     package_data={'': extra_files},
```

Comment:
If you look here https://github.com/wellcometrust/rori/blob/master/setup.py#L23 at how we have entry_points set up on rori, it means that when you install grants_tagger onto a system, it will make an alias in /usr/bin/<name> so that you can run the package as a CLI tool, then you don't need to wrap the library in an outer python script to run it.

You can just to `<entrypoint_name> --<arg> <value>` without having to invoke python or anything.

Isn't necessary now, but worth noting for libs like this which have a CLI interface defined.

Reply:
Great point and will definitely develop that down the line.

Reply:
I am interested to see that too (but got 404 on tori :( ). Is it the same as the flask setup.py?

Reply:
Ah sorry, it just looks like this

```
setup(
    ...
    entry_points={
        "console_scripts": [
            "rori=rori.main:server"
        ]
    },
    ...
)
```

Where you want a CLI tool available to the user as `rori` which maps to the rori{package}.main{pythonfile}:server{method}

When this package is installed, you can start up rori by just entering the command `rori --config /etc/rori/config.yaml` on the command-line.